CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+## v0.24.0 (2024-01-10)
+
+## What's Changed
+
+Release highlights:
+* IP colocation filter can now be changed via a configuration parameter.
+* New filter admin endpoint can now be used to access subscription data.
+
```

Comment:
I'd add the change around the default msg size (and make it configurable in `wakunode2`)